syntax = "proto2";

package protos;
//import "protos/models/user.proto";
import "google/protobuf/timestamp.proto";

message UserCreatedEvent {
    required string event_id = 1;
    optional google.protobuf.Timestamp event_timestamp = 2;
    required string event_name = 3;
    required string version = 4;

//    repeated protos.models.User user = 5;

//    message payload {
//        required protos.models.User user = 1;
//    }
}
